File: ASSIGNMENT/PYTHON_/Fundamentals of python/q5.py
# Write a Python program to find whether a given number is even or odd, print out an appropriate message to the user. 
while(1):
     num=int(input("enter a number:"))
     
     if num %2==0:
        print(num,"is even number")
     else:
        print(num,"is odd number")  

     choice=input("do you want check again(y/n):").lower()  
     if(choice!='y'):
       break
print("thank you")     

File: ASSIGNMENT/C_/loop/q5.c
//q5
//WAP to print factorial of given number
#include<stdio.h>

int main(){

    int i, n, f=1;// factorial
    printf(" enter a num");
    scanf("%d", &n);

    for(i=1;i<=n;i++){

        f = f * i;
        
        printf(" factorial is %d :%d\n",n ,f);
    }

    return 0;
}
